docker-compose.yml

```diff
@@ -5,11 +5,11 @@ server:
   volumes:
    - .:/src
    - /var/run/docker.sock:/tmp/docker.sock
-   - ./config.yml:/config.yml
+   - ~/volumes/ci/roger:/roger/
```

Comment:
again, this will only work if you actually have that directory in your home :) I think we could actually just use the `./config` folder in the root of the repo and move the `config.yml` there